TrackingAntennaV3/main/Accelerometer.hpp

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file   Magnetometer.hpp
+ * @Author Michael Lenover (WARG)
+ * @date   January 25, 2017
+ * @brief  
+ * @see https://www.arduino.cc/en/Reference/Ethernet
+ */
+
+#ifndef ACCELEROMETER
+#define ACCELEROMETER
+
+#define COMPASS_ID 12345
+
+//Accelerometer data struct
+typedef struct {
```

Comment:
Specifiy the units in the comment for this struct. Also use the /** **/ type of comments that you can see in my PR's when defining structs, preprocessor definitions, or functions